tests/nlp/checks/train_test_validation/label_drift_test.py

```diff
@@ -63,9 +81,7 @@ def test_multi_label_without_drift(dummy_multilabel_textdata_train_test):
     # Assert
     assert_that(condition_result, has_items(
         equal_condition_result(is_pass=True,
-                               details="Label's drift score Cramer's V is 0",
+                               details="Label's drift score Cramer's V is 0.02",
```

Comment:
why did this change?

Reply:
cause the previous calculation method was bad